src/intl/en/page-developers-docs.json

```diff
@@ -131,5 +131,13 @@
   "page-calltocontribute-link": "documentation template",
   "page-calltocontribute-link-2": "Discord server",
   "page-calltocontribute-span": "Edit page",
-  "page-calltocontribute-title": "Help us with this page"
+  "page-calltocontribute-title": "Help us with this page",
+  "layer-2-arbitrum-note": "Fraud proofs only for whitelisted users, whitelist not open yet",
+  "layer-2-boba-note": "State validation in development",
+  "layer-2-metis-note": "Fraud proofs in development",
+  "layer-2-optimism-note": "Fault proofs in development",
```

Comment:
I think this PR needs these translation files strings looked at. Some of these files don't make sense for some of these strings. Let me know if you want to sort through this together

Reply:
Note this is in a few json files, just calling out the pattern.

Reply:
Not sure if I follow you on this. LMK and we can get on a call.

Reply:
Ignore this, I understand why this is here now after a quick chat.